django_heroku/core.py

```diff
@@ -43,6 +43,19 @@ def teardown_databases(self, old_config, **kwargs):
 
 
 def settings(config, *, db_colors=False, databases=True, test_runner=True, staticfiles=True, allowed_hosts=True, logging=True, secret_key=True):
+    if 'USE_MAX_CONN_AGE' in os.environ:
+        use_max_conn_age = bool(os.environ['USE_MAX_CONN_AGE'])
```

Comment:
Most people want `1` to mean `True` and `0` to mean `False` but this will always evaluate to `True` regardless of the following values:

- `USE_MAX_CONN_AGE=1`
- `USE_MAX_CONN_AGE=0`
- `USE_MAX_CONN_AGE=True`
- `USE_MAX_CONN_AGE=False`
- `USE_MAX_CONN_AGE=false`
- `USE_MAX_CONN_AGE=fALSE`
- `USE_MAX_CONN_AGE=not-a-bool`

Reply:
@sigmavirus24 @burcaw What if the line was:




```python
try:
   # already defined in settings before
   MAX_CONN_AGE
except NameError:
    # assuming no preference. set a sensible default
    # 600 doesn't impact local and makes heroku's postgres servers happier
   MAX_CONN_AGE = 600
 ``` 

tbh, this setting I used to update in my setting_prod.py with `DATABASES['defaut']['CONN_MAX_AGE'] = 600`. django's default of 0 is rather silly and adds extra connection overhead without people knowing. For most indivudals not a big deal, but across an entire infrastructure I can see why these opionated defaults are in place.